src/styles/blocks/cards.scss

```
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white;
+  padding: 32px 16px 16px;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent;
+    margin: 0;
+    margin-bottom: 4px;
+  }
+
+  &__image {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
+
+  .card__title {
+    color: #34568b;
+  }
+}
```

Need to fix it everywhere
```suggestion
.card {
  display: flex;
  flex-direction: column;

  border-radius: 5px;
  border: 1px solid #f3f3f3;
  background-color: $white;
  padding: 32px 16px 16px;

  &__title {
    font-size: 12px;
    font-weight: 500;
    line-height: 18px;
    color: $main-accent;
    margin: 0;
    margin-bottom: 4px;
  }

  &__image {
    margin: 0 auto 40px;
    width: 160px;
    height: 134px;
  }

  &__code {
    font-size: 10px;
    line-height: 14px;
    color: $secondary;
    margin: 0;
    margin-bottom: 16px;
  }
  
  &:hover {
    &__title {}
  }
}
```